packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -1,31 +1,68 @@
 import React from 'react';
-import { Tabs } from '@deriv/components';
+import { Tabs, Tab } from '@deriv/components';
 import { localize } from '@deriv/translations';
+import Chart from 'Components/chart';
+import DashboardComponents from './dashboard-components';
+import ReactJoyride from 'react-joyride';
+import JoyrideConfig from './joyrideConfig';
+import BotBuilder from './bot-builder';
+import classNames from 'classnames';
+import RunStretegy from '../toolbar/runStrategy';
+import { connect } from 'Stores/connect';
+import RootStore from 'Stores/index';
+import Sidebar from './dashboard-components/sidebar';
 
-const Dashboard: React.FC = () => {
-    const [active_index, setActiveTabIndex] = React.useState(0);
+interface DashboardProps {
+    active_tab: number;
+    setActiveTab: (active_tab: number) => void;
+}
+
+const Dashboard = ({ active_tab, setActiveTab }: DashboardProps) => {
+    const [showSideBar, setshowSideBar] = React.useState<boolean>(true);
+    const { DashBorardSteps } = JoyrideConfig;
+    const [tourRun, setTourRun] = React.useState<boolean>(true);
+    const handleClick = (e: any) => {
+        e.preventDefault();
+        setTourRun(true);
+    };
 
     return (
-        <div className='dashboard__container'>
-            <Tabs active_index={active_index} onTabItemClick={setActiveTabIndex} top>
-                {/* [Todo] needs to update tabs component children instead of using label property */}
-                <div label={localize('Dashboard')}>
-                    <div>
-                        <h1>Create or start a bot</h1>
-                    </div>
-                </div>
-                <div label='Quick Strategy'>
-                    <div>Contennt 2</div>
-                </div>
-                <div label='Bot Builder'>
-                    <div>Contennt 3</div>
-                </div>
-                <div label='Tutorial'>
-                    <div>Contennt 4</div>
+        <div className='main_dashboard_container'>
+            <div className={classNames('dashboard__container', { 'w-100': !showSideBar })}>
+                <div className='dashboard__run-stretegy-wrapper'>
+                    <RunStretegy />
                 </div>
-            </Tabs>
+                <ReactJoyride steps={DashBorardSteps} run={tourRun} continuous={true} showProgress={true} />
+                <Tabs active_index={active_tab} onTabItemClick={setActiveTab} top>
+                    {/* [Todo] needs to update tabs comIcDashBoardComponentsTabponent children instead of using label property */}
+                    <Tab icon='IcDashboardComponentTab' label={localize('Dashboard')}>
+                        <DashboardComponents />
+                    </Tab>
+                    <Tab icon='IcBotbuilderTabIcon' label='Bot Builder' id='id-bot-builder'>
```

Comment:
ditto - localize
On all the other labels too

Reply:
done